src/metrics/matcher/ruleset.go

```diff
@@ -195,6 +197,14 @@ func (r *ruleSet) toRuleSet(value kv.Value) (interface{}, error) {
 	if err := value.Unmarshal(r.proto); err != nil {
 		return nil, err
 	}
+	for _, rr := range r.proto.RollupRules {
+		for _, s := range rr.Snapshots {
+			for _, t := range s.TargetsV2 {
+				r.opts.InstrumentOptions().Logger().Info("adding rollup rule",
+					zap.String("name", s.Name), zap.Bool("resendEnabled", t.ResendEnabled))
+			}
+		}
+	}
```

Comment:
Do we still want this? "Info" level logging seems noisy. Perhaps debug level if we want to keep it? (but would opt for dropping completely given this is a library function rather than an API endpoint)